src/views/v2/list/event/title.php

```diff
@@ -15,7 +15,7 @@
 $event    = $this->get( 'event' );
 $event_id = $event->ID;
 ?>
-<h3 class="tribe-events-calendar-list__event-title tribe-common-h4">
```

Comment:
keep this but change to `h5`

Reply:
We are updating the styles. I don't understand why we should change the semantics. Can you please expand a bit more on this request?

Reply:
I don't mean the tag, i meant change to `tribe-common-h5` class. This is connected to https://github.com/moderntribe/the-events-calendar/pull/2560#discussion_r294078672

Reply:
from the call with design, we'll use `tribe-common-h6 tribe-common-h5--min-medium` classes here.